eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -228,6 +234,14 @@ jobs:
       - name: buildPassArguments
         value: ''
 
+    ### Additional arguments
+    - ${{ if eq(parameters.enableIBCOptimization, 'true') }}:
+      - name: ibcArguments
+        value: '/p:EnableIBCOptimization=true /p:IBCDropAccessToken=$(dn-bot-devdiv-drop-rw-code-rw)'
```

Comment:
for my understanding, why does this need code r/w if we're just restoring packages? or do we just not have a more fine-grained token?

Reply:
We don't have a more fine grained token...let me file an issue to fix that.

Reply:
https://github.com/dotnet/arcade/pull/15481